st2actions/st2actions/notifier/notifier.py

```diff
@@ -231,11 +226,25 @@ def _get_trace_context(self, execution_id):
         return None
 
     def _post_generic_trigger(self, liveaction_db=None, execution_db=None):
-        if not ACTION_SENSOR_ENABLED:
+        if not cfg.CONF.action_sensor.enable:
             LOG.debug('Action trigger is disabled, skipping trigger dispatch...')
             return
 
         execution_id = str(execution_db.id)
+        extra = {'execution': execution_db}
+
+        target_statuses = cfg.CONF.action_sensor.emit_when
```

Comment:
Small style thing - I would do `target_statuses = cfg.CONF.action_sensor.emit_when or LIVEACTION_COMPLETED_STATES` this way we could get rid of else clause.

Besides that, LGTM :+1: